operations/github/onboarding/onboarding.sh

```diff
@@ -237,7 +245,7 @@ else
     handle_error "Missing data: parent team name for $MAIN_TEAM"
 fi
 
-# Create  main team
+# Create  main team 
```

Comment:
Do *NOT* allow trailing whitespace - that can create mystery problems & "changes" later.

Also, why 2 spaces between "Create" and "main". Should be 1.